### PR TITLE
Add page entries info to serials page title to distinguish titles

### PR DESCRIPTION
**Pull request by myokoym**
Close https://github.com/bm-sms/nursepress/issues/284

### CODE REVIEW

**Comment by okkez** [comment]
Looks nice :sunglasses:

File: app/decorators/site_decorator.rb [hunk 1]
```diff
@@ -34,4 +34,13 @@ def page_title(posts)
       tagline
     end
   end
+
+  def serials_page_title(serials)
```
Comment (tricknotes) [inline app/decorators/site_decorator.rb]: Can you create `SerialsDecorator` and move `serials_page_title` to it?
The `serials_page_title` method doesn't use the attributes of `Site`, so this method shouldn't exist on `SiteDecorator`.

Comment (myokoym) [inline app/decorators/site_decorator.rb]: It's reasonable. I'll create `SerialsDecorator`.

Comment (myokoym) [inline app/decorators/site_decorator.rb]: I've fixed it by 3f15bdae03863425419db94e6c5599f1a22e6f2a .

File: app/controllers/serials_controller.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
+    @serials.extend(PaginationInfoDecorator)
+    @serials.extend(SerialsDecorator)
```
Comment (tricknotes) [inline app/controllers/serials_controller.rb:4]: I think `SerialsDecorator` depends on `PaginationInfoDecorator`. ( https://github.com/bm-sms/daimon-news/pull/551/files#diff-f7ddda9786b3de2ac334a4bc6f8d5751R5 👀 )
Can you include `PaginationInfoDecorator` into `SerialsDecorator`?

File: app/controllers/serials_controller.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
```
Comment (cafedomancer) [inline app/controllers/serials_controller.rb:3]: By the way, it is not related to this pull request, but what is `order(id: :desc)`? I don't know its intention from this code.

Comment (okkez) [inline app/controllers/serials_controller.rb:3]: In SQL specification, unordered query returns unsorted result possibly.
So we have to specify order, if we need stable sorted result.

Comment (cafedomancer) [inline app/controllers/serials_controller.rb:3]: OK. Give it an appropriate name as `scope`? It's the problem that we don't understand that from `order(id: :desc)`.

Comment (okkez) [inline app/controllers/serials_controller.rb:3]: - Pagination needs sorted list (result of query)
- Seiral model does not have `order` column

---

:memo: 連載は新しい順に並んでるのが自然だと思う。

File: app/controllers/serials_controller.rb (hunk 2 above)
Comment (myokoym) [inline app/controllers/serials_controller.rb:4]: I understand. I've fixed it.

File: app/controllers/serials_controller.rb (hunk 3 above)
Comment (tricknotes) [inline app/controllers/serials_controller.rb:3]: We discussed on chat about this topic.
We decided name this order specification as scope and use it here.
- To specify order for pagination.
- To be meaningful for programmer.

**Comment by tricknotes** [comment]
Looks awesome!
Thanks for your nice fix @myokoym 🚢

**Pull request merged**